metricbeat/module/elasticsearch/elasticsearch.go

```diff
@@ -368,61 +367,6 @@ func GetXPack(http *helper.HTTP, resetURI string) (XPack, error) {
 	return xpack, err
 }
 
-type boolStr bool
-
-func (b *boolStr) UnmarshalJSON(raw []byte) error {
```

Comment:
was this only used by `hidden` flag ?

Reply:
yes. I couldn't find other usages of that function